src/options.rs

```diff
@@ -23,7 +23,7 @@ use crate::column::Salt;
 use crate::compress::CompressionType;
 use rand::Rng;
 
-const CURRENT_VERSION: u32 = 4;
+pub const CURRENT_VERSION: u32 = 4;
 const LAST_SUPPORTED_VERSION: u32 = 3;
```

Comment:
```suggestion
// TODO on last supported 4, remove `ValueTable` `no_compression` field.
const LAST_SUPPORTED_VERSION: u32 = 3;
```
or similar comment to not forget it.